Packs/Salesforce/Integrations/SalesForceEventCollector/SalesForceEventCollector.yml

```diff
@@ -0,0 +1,92 @@
+commonfields:
+  id: Salesforce Event Collector
+  version: -1
+name: Salesforce Event Collector
+display: Salesforce Event Collector
+category: Analytics & SIEM
+description: Salesforce logs event collector integration for XSIAM.
+configuration:
+- display: Server URL
+  name: url
+  defaultvalue: 'https://um6.salesforce.com'
+  type: 0
+  required: true
+- display: ""
+  displaypassword: Client ID
+  name: client_id
+  type: 9
+  required: true
+  hiddenusername: true
+- display: ""
+  displaypassword: Client Secret
+  name: client_secret
+  type: 9
+  required: true
+  hiddenusername: true
+- display: Username
+  name: username
+  type: 0
+  required: true
+- display: ""
+  displaypassword: Password
+  name: password
+  type: 9
+  required: true
+  hiddenusername: true
```

Comment:
Does the user specifically to the password?
If so, they should be a single parameter.